app/views/reactions/_list.html.erb

```diff
@@ -22,6 +22,9 @@
     <% end %>
 </div>
 
+<%# TODO: eager load instead %>
```

Comment:
Is this something we need to address in this PR, or a note for later?  (Fine with me either way; I just didn't know whether to block on it, so I'm just commenting.)

Reply:
Just a note for later, I tend not to add TODOs unless it's critical to fix before updating. This one in particular is just an observation that we can remove `host` hoisted out of the loop if we preload `community` (otherwise, it'll result in N+1 queries)